includes/classes/wp-maintenance-mode.php

```diff
@@ -1293,6 +1284,59 @@ public function send_contact() {
 			}
 		}
 
+		/**
+		 * Save subscriber into database.
+		 *
+		 * @param Form_Data_Request $form_data The form data.
+		 * @return void
+		 */
+		public function otter_add_subscriber( $form_data ) {
+			if ( $form_data ) {
+				$input_data = $form_data->get_payload_field( 'formInputsData' );
+				$input_data = array_map(
+					function( $input_field ) {
+						if ( isset( $input_field['type'] ) && 'email' === $input_field['type'] ) {
+							return $input_field['value'];
+						}
+						return false;
+					},
+					$input_data
+				);
+				$input_data = array_filter( $input_data );
+				if ( ! empty( $input_data ) ) {
+					foreach ( $input_data as $email ) {
+						$this->insert_subscriber( $email );
+					}
+				}
+			}
+		}
+
+		/**
+		 * Save subscriber into database.
+		 *
+		 * @param string $email Email address.
+		 * @global object $wpdb
+		 *
+		 * @return void
+		 */
+		public function insert_subscriber( $email = '' ) {
+			global $wpdb;
+			if ( ! empty( $email ) ) {
+				// phpcs:ignore WordPress.DB.DirectDatabaseQuery
+				$exists = $wpdb->get_row( $wpdb->prepare( "SELECT id_subscriber FROM {$wpdb->prefix}wpmm_subscribers WHERE email = %s", $email ), ARRAY_A );
+				if ( empty( $exists ) ) {
+					// phpcs:ignore WordPress.DB.DirectDatabaseQuery
+					$wpdb->insert(
+						$wpdb->prefix . 'wpmm_subscribers',
+						array(
+							'email'       => $email,
```

Comment:
@girishpanchal30 make sure to sanitize email using `sanitize_email`

Reply:
@selul Done! Thank you for the quick review.